{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -283,6 +283,9 @@
 }
 
 REDIS_URL = env("REDIS_URL", default="redis://{% if cookiecutter.use_docker == 'y' %}redis{%else%}localhost{% endif %}:6379/0")
+CELERY_BROKER_USE_SSL = env.bool("CELERY_BROKER_USE_SSL", default=False)
+CELERY_REDIS_BACKEND_USE_SSL = env.bool("CELERY_REDIS_BACKEND_USE_SSL", default=False)
```

Comment:
Maybe we can simplify things?

```suggestion
CELERY_BROKER_USE_SSL = env.bool("REDIS_SSL", default=False)
CELERY_REDIS_BACKEND_USE_SSL = env.bool("REDIS_SSL", default=False)
```